package.json

```diff
@@ -43,6 +43,12 @@
       "type": "string",
       "default": ""
     },
+    "localNodeModules": {
+      "title": "Path to the local node_modules folder",
+      "description": "Optionally specify the path to the local node_modules folder",
```

Comment:
Maybe prefix this with `Advanced: `? Something to indicate this is an uncommon feature and is likely something the user doesn't need except in very specific circumstances.